waspc/cli/src/Wasp/Cli/Command/Build.hs

```diff
@@ -61,6 +63,32 @@ build = do
     else
       throwError $
         CommandError "Building of wasp project failed" $ show (length errors) ++ " errors found"
+  where
+    -- Until we implement the solution described in https://github.com/wasp-lang/wasp/issues/1769,
+    -- we're copying all files and folders necessary for the build into the .wasp/build directory.
+    -- We chose this approach for 0.12.0 (instead of building from the project root) because:
```

Comment:
Yes, precisely because of this hack here: https://github.com/wasp-lang/wasp/blob/52a1910a74b961b1d16cad443718b16d9df58eef/waspc/src/Wasp/Generator/SdkGenerator/Common.hs#L52